compendium/DeclarativeServices/src/SCRLogger.cpp

```diff
@@ -126,5 +126,20 @@ namespace cppmicroservices
                 currLogger->Log(sr, level, message, ex);
             }
         }
+
+	std::shared_ptr<cppmicroservices::logservice::Logger>
+        SCRLogger::getLogger(const std::string& name) const
+	{
+	    auto currLogger = std::atomic_load(&logService);
+	    return currLogger->getLogger(name);
+	}
+
+	std::shared_ptr<cppmicroservices::logservice::Logger>
+        SCRLogger::getLogger(const cppmicroservices::Bundle& bundle, const std::string& name) const
+        {
+            auto currLogger = std::atomic_load(&logService);
```

Comment:
a crash will happen if `currLogger` is `nullptr`. This needs to be handled gracefully.